rover_common/src/heartbeatlib.py

```diff
@@ -53,7 +53,7 @@ def heartbeat_handler(self, channel, data):
         self.lcm_.publish(self.where, ret_msg.encode())
 
 
-class OnboardHeartbeater(Heartbeater):
+class JetsonHeartbeater(Heartbeater):
```

Comment:
JetsonHeartBeater

Reply:
Not going to readme but I can add looking into this in the readme